### DISCUSSION TITLE
Bank Option: Allowing individual selling among other things.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#8923] [Ideas and suggestions]
With the current crew wallet implementation crewmembers can now individually purchase items with their money, but to earn money to buy they either have to get paid salary or constantly request money from the bank whenever they have to buy something.

If crew can now buy items on their own, its a logical extension that they can be allowed to sell items on their own as well outside of the standard permission required to buy/sell from the bank's money. This makes sense to add as an additional permission option whenever individual crew wallets are enabled.
 _(There could also be option to allow crew selling - but still go into the shared salary instead of crew wallet, so the whole sub benefits from profits)_

When buying or selling is not enabled, some option to streamline the process of asking for money from the bank or submitting items to be sold would be beneficial as well. 
For instance: the ability to add/remove items from a list which functions same as the buying and selling GUI - and this list of requests can be approved by anyone who has the correct manage-bank or buy/sell permissions.

**Comment by Kaillera** [comment]
My workaround so far is to just give SellInventory permission in the preset in data folder as a default.
